test/unit/lib/plugins/create/create.test.js

```diff
@@ -9,9 +9,11 @@ const Serverless = require('../../../../../lib/Serverless');
 const sinon = require('sinon');
 const walkDirSync = require('../../../../../lib/utils/fs/walkDirSync');
 const download = require('../../../../../lib/utils/downloadTemplateFromRepo');
+const { ServerlessError } = require('../../../../../lib/classes/Error');
```

Comment:
let's import from `lib/serverless-error` instead of `lib/classes/Error` - `Error` module is going to be deprecated at some point in the future